Sources/Subs-Boards.php

```diff
@@ -1312,7 +1312,7 @@ function getBoardTree()
 		'b.description', 'b.child_level', 'b.board_order', 'b.count_posts', 'b.member_groups',
 		'b.id_theme', 'b.override_theme', 'b.id_profile', 'b.redirect', 'b.num_posts',
 		'b.num_topics', 'b.deny_member_groups', 'c.id_cat', 'c.name AS cat_name',
-		'c.description AScat_desc', 'c.cat_order', 'c.can_collapse'
+		'c.description AS cat_desc', 'c.cat_order', 'c.can_collapse',
```

Comment:
No. The newlines and commas here are merely separating the elements of the $boardColumns array. The newlines and commas are not part of any of the array elements themselves, so they will not be part of the imploded string.